### DISCUSSION TITLE
Instruction response reasoning... how to remove

### DISCUSSION THREAD

**Discussion by martinellison** [LostRuins/koboldcpp#750] [Q&A]
Newbie to koboldcpp. I am using the mistral model. It is giving me lots of lines in explanation such as reasoning. Also it is giving me some image links that do not work.

Is there any way to disable this?

(Edited) examples of what I am getting (and do not want):
```
Instruction:\n ...
Reasoning for this response:\nThis is also where I added the torch object ...
Response:\n![image](https://user-images.githubusercontent.com/103874695/206194707-c91ec017-7414-40c6-8408-3921dbbd0d9d.png)...
```

**Comment by LostRuins** [comment]
The model is hallucinating. Try a different model? 
I recommend this one for low end GPUs: https://huggingface.co/Sao10K/Fimbulvetr-11B-v2-GGUF/resolve/main/Fimbulvetr-11B-v2.q3_K_S.gguf

**Reply by martinellison** [reply]
I'm downloading it thanks. *Edit: yes, your model is working properly and my adventurers have already killed a basilisk.*

I am using an RTX 3060 with 12GB of VRAM -- does that count as 'low end'?

**Reply by LostRuins** [reply]
That should work quite nicely. You should be able to do a full offload
